sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -94,6 +94,80 @@ public static void validateUpdateContent(Request contentRequestDto) {
               ERROR_CODE);
         }
       }
+      List<Map<String, Object>> assessmentData = (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.ASSESSMENT_EVENTS);
+      if (!CollectionUtils.isEmpty(assessmentData)) {
+        for (Map<String, Object> map : assessmentData) {
+          if (map.containsKey(JsonKey.COURSE_ID)) {
```

Comment:
please negate the condition and avoid nested if and else conditions
if (!map.containsKey(JsonKey.COURSE_ID) || StringUtils.isBlank((String) map.get(JsonKey.COURSE_ID))){
throw new ProjectCommonException(
                      ResponseCode.courseIdRequired.getErrorCode(),
                      ResponseCode.courseIdRequiredError.getErrorMessage(),
                      ERROR_CODE);
}
please do the same for all other similar validations

Reply:
updated with these changes